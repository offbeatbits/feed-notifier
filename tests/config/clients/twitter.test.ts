import dotenv from 'dotenv';
import path from 'path';

describe('Twitter client config', () => {
  dotenv.config({
    path: path.resolve(process.cwd(), '.env.testing'),
  });

  it('should export Twitter config', () => {
    expect(require('../../../src/config/clients/twitter')).toMatchSnapshot(); // eslint-disable-line
  });
});
