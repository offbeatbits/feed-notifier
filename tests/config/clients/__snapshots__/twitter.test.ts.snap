// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Twitter client config should export Twitter config 1`] = `
Object {
  "twitter": Object {
    "accessToken": "test_twitter_access_token",
    "accessTokenSecret": "test_twitter_access_token_secret",
    "key": "test_twitter_client_key",
    "secret": "test_twitter_client_secret",
  },
}
`;
